src/common/main.lua

```diff
@@ -22,6 +22,7 @@ function ant.init()
 	Denomination = Denomination or 0
 	TotalSupply = TotalSupply or 1
 	Initialized = Initialized or false
+	SourceCodeTxId = SourceCodeTxId or nil
```

Comment:
can we set this to current ANT src code id?

Reply:
Hm. Actually, yeah. That might also prevent use from needing to call eval twice.

Reply:
Its a little funky because we would need to publish one, then use that source code id in the code, then publish that one... and then use the new source code id, which doesnt match the id in the code... eh.... Not a fan, that gets confusing.

Reply:
we could, yeah, but its the same issue. Its nbd, just adds an extra step to deployment. I can automate it easily.